src/base/bittorrent/session.cpp

```diff
@@ -1139,9 +1142,10 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
 #endif
 
     if (m_listenInterfaceChanged) {
-        const ushort port = this->port();
-        const std::pair<int, int> ports(port, port);
-        settingsPack.set_int(libt::settings_pack::max_retry_port_bind, ports.second - ports.first);
+        const int port = useRandomPort() ? 0 : this->port();
```

Comment:
Can `m_port` be used instead of `this->port()`?

Reply:
I would prefer using the getter function. How about I rename the variable?